composer.json

```diff
@@ -75,6 +76,9 @@
             "web/profiles/contrib/{$name}": ["type:drupal-profile"],
             "web/themes/contrib/{$name}": ["type:drupal-theme"],
             "drush/contrib/{$name}": ["type:drupal-drush"]
-        }
+        },
+        "preserve-paths": [
+            "web/sites/development.services.yml"
```

Comment:
Nevermind! Keep this here, looks like Drupal Scaffold is still ignoring the "initial" parameter, even when it's set manually. Thought I had tested it... oh well! Let's do add in `web/sites/default/services.yml` as well, so we can version control that one.